README.md

```diff
@@ -1,33 +1,50 @@
 # NPI_FILES
+
 repo to automate NPI file download, extract and load into a database and create an api for endpoint
 
-## Podman setup in script  
+## Podman setup in script
 
 work in progress... `pod_setup.sh`
 
-## Local Development for Django  
+## Local Development for Django
+
+Need to set up Postgres, either locally or in a container. The container is my choice as it makes it easier when testing with the containerized django app. The instructions below are specific to local Postgres environment.
+
+## Prerequisite
+
+This requirement will have you update the `.env.dummy` file with appropriate values and rename it to `.env` so you don't check in any passwords or settings. These values are referenced in Django's `settings.py` file
+
+1. Navigate to `NPI_FILES/backend`
+
+2. Create a copy of `.env.dummy`
+
+3. Update all the relevant fields with your local postgres.
+
+```
+  #Sample
+  POSTGRES_USER=postgrestestuser
+  POSTGRES_PASSWORD=postgrespassword
+  POSTGRES_SERVER=localhost
+  POSTGRES_DB=postgresdb
+  POSTGRES_PORT=1234
```

Comment:
lets just use the stuff in the file, this makes it easier to use to see if someone even changed these settings or if they db is already these settings.